torchvision/prototype/datasets/utils/_dataset.py

```diff
@@ -42,7 +42,7 @@ def __iter__(self) -> Iterator[Dict[str, Any]]:
         yield from self._dp
 
     @abc.abstractmethod
-    def _resources(self) -> List[OnlineResource]:
+    def _resources(self) -> Sequence[OnlineResource]:
```

Comment:
This gets rid of the annoying `mypy` warning when returning `List[ManualDownloadResource]` Instead of `List[OnlineResource]`.